rasa/cli/arguments/train.py

```diff
@@ -101,6 +102,30 @@ def add_compare_params(
     )
 
 
+def add_dry_run_param(
+    parser: Union[argparse.ArgumentParser, argparse._ActionsContainer]
+) -> None:
+    """Add `--dry-run` argument to a specified `parser`.
+
+    Args:
+        parser: An instance of `ArgumentParser` or `_ActionsContainer`.
+    """
+    parser.add_argument(
+        "--dry-run",
+        default=False,
+        action="store_true",
+        help="If enabled, not actual training will be performed. Instead, "
+        "it will be determined whether a model should be re-trained "
+        "and this information will be printed as the output. The return "
+        "code is a 4-bit bitmask that can also be used to determine what exactly needs "
+        "to be retrained:\n"
+        "- 0b0001 means Core needs to be retrained\n"
```

Comment:
Let's just tell them the integer which the bitmask translates to. No need to refresh their knowledge on binary numbers 😂